src/HDF5.jl

```diff
@@ -1924,6 +1925,19 @@ function get_chunk(dset::Dataset)
     ret
 end
 
+# Set a single filter
+function set_filter(p::Properties, filter_id, flags, cd_values...)
+    h5p_set_filter(p::Properties, filter_id, flags, length(cd_values), Ref(Cuint.(cd_values)))
```

Comment:
```suggestion
    h5p_set_filter(p::Properties, filter_id, flags, length(cd_values), Cuint[cd_values...])
```

Reply:
Looks like for some reason 1.3.1 has issues with the `unsafe_convert` of the Tuple to Ptr{UInt32}

Reply:
I'm guessing this was added in https://github.com/JuliaLang/julia/pull/34199

Reply:
Yeah, looks like that was officially added in 1.5: https://github.com/JuliaLang/julia/blob/release-1.5/NEWS.md

What to do?  It would be easy enough to throw an informative exception proactively if someone tries to use this functionality on 1.3.

Reply:
Oh, just seeing your suggestion now.  That will allocate, right?  I wonder if `Ref{Cuint}(cd_values...)` also ends up allocating.

Reply:
The suggested change should work in a backwards compatible way

Reply:
Right, allocation would be a perf issue with the proposed change.

Reply:
Yes, it turns out `Cuint.(cd_values)` allocates more bytes that `Cuint[c_values...]`!  Change pushed.